react-app/ccf/src/index.tsx

```diff
@@ -3,12 +3,16 @@ import ReactDOM from 'react-dom/client';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+import { getStorage } from "firebase/storage";
 import { getAnalytics } from "firebase/analytics";
 //If this line says import not found then you need to set up firebase api on your machine
 import firebaseConfig from "./firebase_config/FireConfig";
 
 
 const cong = initializeApp(firebaseConfig);
+const firestore = getFirestore(cong);
```

Comment:
This firestore variable is called "db" on the main branch, so it might be easier to switch that name now before dealing with the merge conflicts.